pages/treeherder.py

```diff
@@ -13,9 +14,16 @@
 
 class TreeherderPage(Base):
 
+    _clear_all_menu_locator = (By.CSS_SELECTOR, 'ul.dropdown-menu:nth-child(3) > li:nth-child(4) > a:nth-child(1)')
```

Comment:
This looks like a fragile locator. First, I think we should anchor this to `pinboard-controls`. Ideally, we would get a class added to this `li` or `a`, but for now we'll have to rely on the position. Something like `#pinboard-controls .dropdown-menu li:nth-child(4)` would be better.